src/utils.ts

```diff
@@ -70,17 +65,17 @@ export function isTrueJoinEvent(event: any): boolean {
     return membership === 'join' && prevMembership !== "join";
 }
 
-export async function redactUserMessagesIn(mjolnir: Mjolnir, userIdOrGlob: string, targetRoomIds: string[], limit = 1000) {
+export async function redactUserMessagesIn(client: MatrixClient, managementRoom: ManagementRoomOutput, userIdOrGlob: string, targetRoomIds: string[], limit = 1000, noop = false) {
```

Comment:
Documentation would be useful :)

In particular, what's `limit`, what's `userIdOrGlob`, what's `noop`?